### PR TITLE
Release 2.8.1

### PR DESCRIPTION
**Pull request by sergiou87**
## Description

Looking for the PR for the upcoming v2.8.1 production release? Well you've just found it, congratulations! :tada:

This is based on the 2.8.0 release (tag [release-2.8.0](https://github.com/desktop/desktop/releases/tag/release-2.8.0)) and includes:
- Fix for crash reporting crashes #12089 
- Fix for crash showing crash to users #12104 
- Disable partial commits while hiding whitespace changes #12129

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release 😂  **LOL** 🤣  ALL LOOKS GOOD, KTHXBAI
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

**Review by tidy-dev — DISMISSED** [review]
✨

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "releases": {
+    "2.8.1": [
+      "[Fixed] Disable partial change selection in split view while whitespace changes are hidden."
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Disable partial change selection in split view while whitespace changes are hidden - #12129"
```

**Review by billygriffin — DISMISSED** [review]
🚀 🤘 🎸

File: app/src/ui/diff/side-by-side-diff-row.tsx [hunk 2]
```diff
@@ -326,7 +330,7 @@ export class SideBySideDiffRow extends React.Component<
 
     return (
       <div
-        className="hunk-expansion-handle selectable"
+        className="hunk-expansion-handle selectable hoverable"
```
Comment (sergiou87) [inline app/src/ui/diff/side-by-side-diff-row.tsx:333]: I just added this `hoverable` here, cherry-picked from #12138

**Comment by sergiou87** [comment]
I've updated the branch with the latest `development` to resolve the conflicts before merging, but only what you saw yesterday (+ today's fixes from #12138) has been released:

image

**Review by tidy-dev — APPROVED** [review]
✨

**Pull request merged**